src/shared/components/card/cardProduct/CardProduct.css.ts

```
@@ -111,3 +134,109 @@ export const brandText = style({
+  paddingTop: '1.2rem',
+  gap: '0.8rem',
+});
+
+export const colorRow = style({
+  display: 'flex',
+  alignItems: 'center',
+  gap: '0.2rem',
+});
+
+export const colorChip = style({
+  width: '1.4rem',
```

ESLint

[error] Replace `'1.4rem'` with `"1.4rem"`

(prettier/prettier)